### PR TITLE
Sentry appflow

### PR DESCRIPTION
**Pull request by iravisankar**

### CODE REVIEW

File: build_appflow.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+VERSION=`sentry-cli releases propose-version`
+ionic capacitor sync --prod --source-map
```
Comment (iamtarun1993) [inline build_appflow.sh:2]: are you sure this was the command??

Comment (iamtarun1993) [reply build_appflow.sh:2]: command was 
ionic capacitor build --prod --source-map
ionic capacitor sync

File: src/main.ts [hunk 2]
```diff
@@ -11,7 +11,7 @@ import 'hammerjs';
 Sentry.init({
   dsn: environment.SENTRY_DSN,
   tracesSampleRate: 1.0,
-  release: 'please-replace-your-git-commit-version',
+  release: 'undefined',
```
Comment (iamtarun1993) [inline src/main.ts:14]: why this become undefined ??

File: hooks/secrets.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 const dotenv = require('dotenv');
 dotenv.config();
 const NATIVE_CONFIG = JSON.parse(process.env.NATIVE_CONFIG);
-
+console.log(process.env.CI_GIT_COMMIT_SHA);
```
Comment (iamtarun1993) [inline hooks/secrets.js:4]: do you need to print this?
or it is just for reference to see in app flow terminal so later we can debug if anything got broken.

**Review by Julias0 — APPROVED** [review]

**Review by Dimple16 — APPROVED** [review]

File: build_appflow.sh (hunk 1 above)
Comment (iravisankar) [reply build_appflow.sh:2]: `ionic capacitor build --prod --source-map ` we need mention android or ios for adding build ,
`ionic capacitor sync   --prod --source-map ` will take care of it for both android and ios build.

**Review by Julias0 — APPROVED** [review]

**Pull request merged**